topics/imaging/tutorials/omero-suite/tutorial.md

```diff
@@ -0,0 +1,392 @@
+ROI tools. In this way, it is possible to associate a sequence of annotations to several uploaded images.
+
+### Metadata and ROIs input
+
+The last section is to upload metadata and ROIs
+
+> <hands-on-title> Add tools to upload metadata and ROIs </hands-on-title>
+>
+>1. Add {% tool [OMERO Metadata Import](https://toolshed.g2.bx.psu.edu/view/ufz/omero_metadata_import/5.18.0+galaxy3) %}  from the list of tools:
+>   - Connect the output of {% icon param-file %} **5: OMERO instance address** to the "OMERO host url"
+>   input of {% icon tool %} **12: OMERO Metadata Import**.
+>   - Connect the output of {% icon param-file %} **11: Parse parameter value** to the "Object ID"
+>   input of {% icon tool %} **12: OMERO Metadata Import**.
+>   - Connect the output of {% icon param-file %} **6: Annotation type** to the "Annotation type"
+>   input of {% icon tool %} **12: OMERO Metadata Import**.
+>   - Connect the output of {% icon param-file %} **3: Input Metadata File** to the "Annotation file"
+>   input of {% icon tool %} **12: OMERO Metadata Import**.
+>2. Add {% tool [OMERO ROI Import](https://toolshed.g2.bx.psu.edu/view/ufz/omero_roi_import/5.18.0+galaxy3) %}  from the list of tools:
+>   - Connect the output of {% icon param-file %} **5: OMERO instance address** to the "OMERO host url"
+>   input of {% icon tool %} **13: OMERO ROI Import**.
+>   - Connect the output of {% icon param-file %} **2: Input Tabular ROIs** to the "Tab File with ROIs"
+>   input of {% icon tool %} **13: OMERO ROI Import**.
+>   - Connect the output of {% icon param-file %} **11: Parse parameter value** to the "Image ID where annotate the ROIs"
+>   input of {% icon tool %} **13: OMERO ROI Import**.
+>3. Save your workflow and name it `OMERO_basic_upload`. {: .hands_on}
+
+
+You are done! The final workflow should  look like this:
+![img.png](../../images/omero-suite/workflow.png)
+
+And you can easily add all parameters during the workflow invocation:
+![img_1.png](../../images/omero-suite/workflow_invocation.png)
+
+This workflow can be easily re-used or turned into a sub-workflow for an image processing
+pipeline.
+
+# Conclusion
+
+In this exercise, you imported images into OMERO using Galaxy. You also learn how to
+import metadata and ROIs, as well as get information such as image ID and annotation file
+from a target OMERO instance.
+
+
+# References
+
+- Allan, C., Burel, JM., Moore, J. et al. OMERO: flexible, model-driven data management for experimental biology. 
+Nat Methods 9, 245–253 (2012). https://doi.org/10.1038/nmeth.1896
+- Wilkinson, M., Dumontier, M., Aalbersberg, I. et al. The FAIR Guiding Principles for Scientific Data Management and
+stewardship. Sci Data 3, 160018 (2016). https://doi.org/10.1038/sdata.2016.18
+
+## Acknowledgement
+
+We acknowledge the support of [**NFDI4BIOIMAGE**](https://nfdi4bioimage.de/home/), 
+funded by the German Research Foundation (DFG) within the framework of the NFDI – project numbers: 501864659.
```

Comment:
This is not needed, as you can and have annotated the tutorial with the proper funding and the references can be references via a citations file.